### ISSUE TITLE
Unclear explanation about operator precedence in concepts/conditionals

### ISSUE THREAD

**Issue by Riazaia** [exercism/javascript#1722]
There's this bit of code in the [Conditionals](https://github.com/exercism/javascript/tree/main/concepts/conditionals) part of the syllabus that seems to be missing some brakets so that the explanation makes sense. 

```
if (num >= 0 && num < 1) {
  // ...
}

// The inner brackets are obsolete because relational operators
// have higher precedence than logical operators.
if (num >= 0 && num < 1) {
  // ...
}
```

**Comment by junedev** [comment]
Thanks for creating the issue, I had this on my todo list as well. The inner brackets in the bottom example were removed by the auto-formatting. To make them stick, a prettier ignore block needs to be added.

**Comment by junedev** [comment]
If someone works on this, be aware that there might be multiple places where this same content (and issue) occurs. Please search to check for this and fix it in all the places if necessary.

**Comment by Samhhali** [comment]
Hello All,
I made a search to find any other places where this same content (and issue) occurs, but didn't find other than the issue mentioned, I solved it and made a pull request #1734
